open flexure interview (#25)
open-make/openmakewebsite: content/blog/2023-01-31-interview-open-flexure-microscope/index.en.md

Python. We've also got MATLAB and Python clients that can talk to the
API. The software, I think, is very mature for an open hardware project,
especially a science open hardware project. And I think even if
you consider a lot of very expensive scientific instruments, the
user interface of the Software is often an afterthought. Even for
professional instruments the UI is often terrible. Ours is at least reasonably good. There's a lot of things we need to fix, but it's actually quite reasonable.
{{< /expand >}}

> And has this system been built or produced by others independently?

{{< expand "Show answer">}}
It's very hard to know how many people have built it. You only know the people
who've reported back and told you they've built it. So we did a little
sort of survey asking on Twitter and on our forum. Have you built it? And where
have you built it or used it? So far, it's been used on every continent,
including Antarctica. We've got a photo of somebody using it on the ice
in Antarctica. It's been built in about 40 countries. We also have a user forum with over
250 people. We don't know if every single one of them has built or used

> the microscope, some people will have joined but turn off "ask questions". But the community we know about may just be the tip of the iceberg, we assume there are thousands of microscopes out there,

Author: ```suggestion
the microscope, some people will join only to ask questions. But the community we know about may just be the tip of the iceberg, we assume there are thousands of microscopes out there,
```

Reviewer: I think "will have joined only to ask questions" is most likley what I said/meant to say
